data/processed/ref/mean_ref.yaml

```diff
@@ -0,0 +1,23 @@
+api: https://results.ref.ac.uk
```

Comment:
This is the type of the API, so I think it should be `fetch` or similar because we're fetching a file directly